out/calls-cost.js.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>JSDoc: Source: calls-cost.js</title>
+
+    <script src="scripts/prettify/prettify.js"> </script>
+    <script src="scripts/prettify/lang-css.js"> </script>
+    <!--[if lt IE 9]>
+      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+    <link type="text/css" rel="stylesheet" href="styles/prettify-tomorrow.css">
+    <link type="text/css" rel="stylesheet" href="styles/jsdoc-default.css">
+</head>
+
+<body>
+
+<div id="main">
+
+    <h1 class="page-title">Source: calls-cost.js</h1>
+
+    
+
+
+
+    
+    <section>
+        <article>
+            <pre class="prettyprint source linenums"><code>/**
+ * @typedef {Object} Call
+ * @property {string} identifier - Call's identifier
+ * @property {string} type - Call's type
+ * @property {number} duration - Call's duration
+ */
+
+/**
+ * @typedef {Object} ProcessedCall
+ * @property {string} identifier - Call's identifier
+ * @property {string} type - Call's type
+ * @property {number} duration - Call's duration
+ * @property {number} callCost - Call's cost
+ */
+
+/**
+ * @typedef {Object} CallsResponse
+ * @property {number} totalCalls - Number of processed calls
+ * @property {number} total - Total to pay including all the processed calls
+ * @property {ProcessedCall[]} callsDetails - Processed information about calls
+ */
+
+/** 
+ * Design a solution to calculate what to pay for a set of phone calls. The function must receive an 
+ * array of objects that will contain the identifier, type and duration attributes. For the type attribute, 
+ * the only valid values are: National, International and Local
+ * 
+ * The criteria for calculating the cost of each call is as follows:
+ * 
+ * International: first 3 minutes $ 7.56 -> $ 3.03 for each additional minute
+ * National: first 3 minutes $ 1.20 -> $ 0.48 per additional minute
+ * Local: $ 0.2 per minute.
+ * 
+ * The function must return the total calls, the details of each call (the detail received + the cost of the call) 
+ * and the total to pay taking into account all calls
+ * 
+ * @param {Call[]} calls - Call's information to be processed
+ * 
+ * @returns {CallsResponse}  - Processed information
+*/
+
+const validCallTypes = ['National', 'International', 'Local'];
+
+const filterByType = arrOfObjects => {
+    const result = arrOfObjects.filter(object => validCallTypes.includes(object.type));
+    return result;
+}
+
+const calculateCallCost = obj => {
+    let totalCost = 0;
+    let additionalMinutes = obj.duration - 3;
+    if (obj.type === 'International') {
+        if (obj.duration &lt;= 3) {

```

Comment:
Oops, I hadn't even looked at these! I believe these html files are the output from running jsdoc because I'm quite sure I didn't modify any html files at all (the challenged said not to). It also converts the 'less-than' character as an escape character. Thanks for bringing it up!